packages/esm-patient-programs-app/src/programs/programs-form.scss

```
@@ -33,3 +34,9 @@
   justify-content: space-between;
   height: calc(100vh - 6rem);
 }
+
+.errorMessage {
+  @include type.type-style("label-02");
+  margin-top: 0.5rem;
```

```suggestion
  margin-top: 0.5rem;
  margin-top: spacing.$spacing-03;
```
Remember to import the spacing 
@use "@carbon/styles/scss/spacing";